app/frontend/wallet/helpers/request.ts

```diff
@@ -14,8 +14,8 @@ const request = async function request(url, method = 'GET', body = null, headers
   }
 
   try {
-    const response = await fetch(url, requestParams)
-    if (!response) throw NamedError('NetworkError')
+    const response = await fetch(url, (requestParams as any)) // TS does not like credentials
+    if (!response) throw NamedError('NetworkError','')
```

Comment:
TODO: Figure out if the second parameters goes to UI and if yes, if we don't want some more details there.